addons/dragon/CfgVehicles.hpp

```diff
@@ -0,0 +1,195 @@
+class MainTurret;
+class CfgVehicles {
+    class LandVehicle;
+    
+    class StaticWeapon : LandVehicle {
+        class Turrets {
+            class MainTurret;
+        };
+    };
+    
+    class StaticMGWeapon: StaticWeapon{
+        class Turrets : Turrets  {
+            class MainTurret : MainTurret {
+                class ViewOptics;
+            };
+        };
+    };
+    
+    class AT_01_base_F: StaticMGWeapon {
+        class Turrets : Turrets  {
+            class MainTurret : MainTurret {
+                class ViewOptics: ViewOptics {
+                };
+            };
+        };
+    };
+    
+    class GVAR(dragonStaticBase) : AT_01_base_F {
+        scope = 1;
+        displayName = CSTRING(dragonName);
+        model = QPATHTOF(models\ace_m47_static.p3d);
+        picture = "\A3\Static_F_Gamma\data\UI\gear_StaticTurret_AT_CA.paa";
+        UiPicture = "\A3\Static_F_Gamma\data\UI\gear_StaticTurret_AT_CA.paa";
+        icon = "\A3\Static_F_Gamma\data\UI\map_StaticTurret_AT_CA.paa";
+        threat[] = {0.3,1.0,0.3};
+        cost = 150000;
+        class Damage {
+            tex[] = {};
+            mat[] = {"a3\static_f_gamma\data\staticturret_01.rvmat","a3\static_f_gamma\data\staticturret_01_damage.rvmat","a3\static_f_gamma\data\staticturret_01_destruct.rvmat","a3\static_f_gamma\data\staticturret_02.rvmat","a3\static_f_gamma\data\staticturret_02_damage.rvmat","a3\static_f_gamma\data\staticturret_02_destruct.rvmat","a3\weapons_f_beta\launchers\titan\data\titan_launcher.rvmat","a3\weapons_f_beta\launchers\titan\data\titan_launcher_damage.rvmat","a3\weapons_f_beta\launchers\titan\data\titan_launcher_destruct.rvmat","a3\weapons_f_beta\launchers\titan\data\titan_mtube.rvmat","a3\weapons_f_beta\launchers\titan\data\titan_mtube_damage.rvmat","a3\weapons_f_beta\launchers\titan\data\titan_mtube_destruct.rvmat"};
```

Comment:
Could probably do this
```
mat[] = {
    text1,
    text2,
    text3,
};
```
Would be more readable. Though this doesn't necessarily be readable. not like anyone will want to edit this.